recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -17,3 +17,20 @@ do_install_append() {
     sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
           ${D}${systemd_unitdir}/system/systemd-quotacheck.service
 }
+
+quotacheck_srv = "systemd-quotacheck.service"
+quotaon_srv = "quotaon.service"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+do_install_append () {
+    install -d ${D}${sysconfdir}/systemd/system/multi-user.target.wants
+
+    sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
+         ${D}${systemd_unitdir}/system/${quotacheck_srv}
+
+    sed -i -e "/^.*\/var\/volatile\/log.*$/d" ${D}${sysconfdir}/tmpfiles.d/00-create-volatile.conf
```

Comment:
This change is not related to the changes, presented by this commint

Reply:
accept